packages/fluentui/docs/src/components/ComponentPlayground/ComponentPlaygroundTemplate.tsx

```diff
@@ -19,43 +19,45 @@ const NoopKnobProvider: React.FunctionComponent = props => {
 };
 
 const ComponentPlaygroundTemplate: React.FunctionComponent<ComponentPlaygroundTemplateProps> = props => (
-  <Grid
-    styles={{
-      gridTemplateColumns: '1fr 300px',
-      msGridColumns: '1fr 1rem 300px',
```

Comment:
Ohoh, this breaks layout

![image](https://user-images.githubusercontent.com/14183168/84010223-aa7f6e80-a974-11ea-999a-0f0974420d3d.png)


vs 

![image](https://user-images.githubusercontent.com/14183168/84010212-a3f0f700-a974-11ea-8838-71f339503add.png)

Reply:
Fixed in the best way that keeps the same for chrome and makes it usable for IE